docs/user-guide/share.md

```diff
@@ -1,49 +1,82 @@
-
-<img src="../img/share/share_window_center_zoom.jpg" class="ms-docimage"  style="max-width:400px;"/>
-
-With *Add center and zoom to sharing link*, manually or by clicking the center coordinates with possibility to switch coordinate format. The user can also set *zoom* value from the input field. By default, the center and zoom param are added to the share link with the possibility to add a place marker (when map loads) by selecting the *Add Marker* checkbox. 
+Here the user can copy the resource **URL link** and passed it to a new clipboard or access the shared map through the **QR code**.
 
 ## Social
 
-The social section displays like the following:
+Switch the panel on the social section the user can share the resource on **Facebook**, **Twitter** and **LinkedIn** simply by clicking on the social icon.
 
 <img src="../img/share/social.jpg" class="ms-docimage"  style="max-width:400px;"/>
 
-From here it is possible to share the Resource on Facebook, Twitter and LinkedIn. Also in this case, only opening the window from the maps [Burger Menu](menu-bar.md#burger-menu) the user can add the bbox or center and zoom param.
-
 ## Embed
 
-Accessing the embed section it displays the following: 
+Switch the panel on the embed section the user can copy the **embedded code** or the **APIs**. It is useful, for example, for inserting the resource inside a webpage.
 
 <img src="../img/share/embed.jpg" class="ms-docimage"  style="max-width:400px;"/>
 
-With this section the user can copy the embedded code or the APIs, useful for example when you need to insert the resource inside a webpage. Also in this case, only opening the window from the maps [Burger Menu](menu-bar.md#burger-menu) the user can add the bbox or center and zoom param.
-
 !!!note
     At the moment the embed section is available only for Maps.
+
+## Advanced options
+
+Sharing *maps* and *stories* can be customized through the **Advanced options**. 
+
+!!!note
+    The **Advanced options** are available only opening *Share panel* from the [Burger Menu](menu-bar.md#burger-menu).
+
+### Advanced options for sharing maps
+
+Enable the **Advanced options** from a map *Share panel* the user can include to the URL:
+
+<img src="../img/share/share_window_map_options.jpg" class="ms-docimage"  style="max-width:400px;"/>
+
+* The current **bounding box parameter** of the map visualized by the user
+
+* The center and the zoom of the loaded map by enabling the **Add center and zoom** 
+
+<img src="../img/share/share_window_center_zoom.jpg" class="ms-docimage"  style="max-width:400px;"/>
+
+Related options opens allows the user to:
+
+* Zoom to a specific point on the map by typing the coordinates in two different formats: *Decimal* and *Aeronautical* (That can be chosen through the <img src="../img/button/change-search-tool.jpg" class="ms-docbutton"/> button) or by clicking on the map to grab the coordinates. 
```

Comment:
* Center the shared map to specific coordinates by typing them in two different formats (*Decimal* or *Aeronautical*, that can be chosen through the <img src="../img/button/change-search-tool.jpg" class="ms-docbutton"/> button) or by clicking on the map to set automatically the coordinate fields.